climbing-stairs/HodaeSsi.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def climbStairs(self, n: int) -> int:
+        dp = []
+        dp.append(0)
+        dp.append(1)
+        dp.append(2)
+
+        for i in range(3, n + 1):
+            dp.append(dp[i - 1] + dp[i - 2])
+
+        return dp[n]
```

Comment:
```suggestion
        return dp[-1]
```

파이썬에서는 이렇게 하면 배열 마지막 원소라는 점이 명확해서 가독성에 도움이 됩니다